### ISSUE TITLE
Number of significant digits for arithmetic operations

### ISSUE THREAD

**Issue by t-nishi-24** [esmini/esmini#225]
Hello!
I just happened to find this issue.  

This is a part of the log file when Esmini is run.  
If you look at the last line, you will see that the calculation results have been rounded.  
```
esmini GIT REV: 39f31f9
esmini GIT TAG: v2.20.5
esmini GIT BRANCH: tags/v2.20.5^0
esmini BUILD VERSION: 2.20.5.build.2190
Run simulation decoupled from realtime, with fixed timestep: 0.01
Generated seed 2044050857
Loading test.xosc (v1.1)
OpenDRIVE: test.xodr
Scenegraph: None
Expr ${$Target_Position - $Offset} = 13.88888888888889 - 0.0 = 13.888890
```
The effective number of digits now is 6, but I think we may need more precise calculation results.  
For example, to convert the commonly used km/h to m/s.  
In fact, the simulation results were sometimes different with and without the calculation.

Is it possible to increase the number of significant digits?

**Comment by eknabevcc** [comment]
Yes it's possible. We changed internal representation from float (32 bits) to double (64 bits).
Included in release [v2.20.6](https://github.com/esmini/esmini/releases/tag/v2.20.6).

Please test and report if it's good enough.

**Comment by t-nishi-24** [comment]
Sorry for late reply.  
Using v2.20.7, I checked that the changes are as intended.  
Thank you!!
